src/auth.py

```diff
@@ -4,14 +4,19 @@
 
 
 def get_mediafile_id(meeting_id, path, app, cookie):
-    return meeting_id
-
-    # TODO: Enable the call to the presenter
-    check_request_url = get_check_request_url(meeting_id, path, app)
-    app.logger.debug(f"Send check request: {check_request_url}")
+    presenter_url = get_presenter_url(app, meeting_id, path)
+    app.logger.debug(f"Send check request: {presenter_url}")
+    payload = [
+        {
+            "presenter": "get_mediafile_id",
+            "data": {"meeting_id": meeting_id, "path": path},
+        }
+    ]
 
     try:
-        response = requests.post(check_request_url, headers={"Cookie": cookie})
+        response = requests.post(
+            presenter_url, headers={"Cookie": cookie}, json=payload
```

Comment:
Please pass all headers you get to the presenter service (except content type, content length, ...). There will be extra headers the client will send, that are needed for authentication. Some headers will be set by the HaProxy, that are also needed. Just forwarding them would be sufficient.

Reply:
pass all headers to the presenter, except headers which contain content.